src/fava_portfolio_returns/api/compare.py

```diff
@@ -15,6 +19,22 @@
 class NamedSeries:
     name: str
     data: Series
+    cash_flows: list[tuple[str, float]] | None = None  # [(date, amount)], amount>0 for buy, <0 for sell
+
+
+def get_series_cash_flows(fp: FilteredPortfolio, start_date: datetime.date, end_date: datetime.date):
+    """Get filtered cash flows (excluding dividends)"""
+    cash_flows = fp.cash_flows()
+    cash_flows = filter_cash_flows_by_date(cash_flows, start_date, end_date)
+    cash_flows = convert_cash_flows_to_currency(fp.pricer, fp.target_currency, cash_flows)
+
+    # Aggregate by date, excluding dividends
+    daily_flows: dict[datetime.date, Decimal] = defaultdict(Decimal)
+    for flow in cash_flows:
+        if not flow.is_dividend and flow.amount.number is not None:  # Exclude dividends and handle None values
+            daily_flows[flow.date] += flow.amount.number
+
+    return [(date.isoformat(), float(amount)) for date, amount in sorted(daily_flows.items())]
```

Comment:
```suggestion
    return sorted(daily_flows.items(), key=lambda x: x[0])
```

`date` and `Decimal` formatting is already handled by the JSON encoder